matrix-client.el

```diff
@@ -106,6 +106,14 @@ prefer to keep your name visible so you can see what your display
 name is in each room."
   :type 'boolean)
 
+(defcustom matrix-client-save-token nil
```

Comment:
Any reason this isnt true by default? I think this is what most people want :smile:

Reply:
Good question.  Since the token would allow an attacker to impersonate the user if stolen, I decided to err on the side of caution.  I was thinking that, if someone were just trying out this package, not sure if he's going to keep using it, it might be better to not store the token permanently in a plain-text file in his homedir.  On the other hand, the other Matrix clients I've been trying out don't bother to ask first, and we do at least try to set the permissions to prevent other user accounts from accessing it, so maybe we should enable it by default.  What do you think?  :)

Reply:
> Since the token would allow an attacker to impersonate the user if stolen, I decided to err on the side of caution.

Hmm, that's a good point, now I'm a bit conflicted :). I think it's probably a good idea to err on the side of security and not save them by default, but point out that the setting exists (when we revamp the readme at some point). It's possible that other matrix clients are doing something special to ensure the tokens can't be stolen somehow, so that's worth looking into at some point...